new/types/manager-types.yaml

```diff
@@ -31,6 +37,49 @@ node_types:
             task_retry_interval: 30
           policy_engine:
             start_timeout: 30
+      security:
+        description: >
+          Security Configuration for Cloudify Manager
+        default:
+          enabled: true
+          authentication_providers:
+            - name: password
+              implementation: flask_securest.authentication_providers.password:PasswordAuthenticator
+              properties:
+                password_hash: plaintext
+            - name: token
+              implementation: flask_securest.authentication_providers.token:TokenAuthenticator
+              properties:
+                secret_key: my_secret
+          userstore_driver:
+            implementation: flask_securest.userstores.simple:SimpleUserstore
+            properties:
+              userstore:
+                user1:
+                  username: example_user1
+                  password: example_password1
+                  email: example_user1@your_domain.dom
+                user2:
+                  username: example_user2
+                  password: example_password2
+                  email: example_user2@your_domain.dom
+                user3:
+                  username: example_user3
+                  password: example_password3
+                  email: example_user3@your_domain.dom
+              identifying_attribute: username
+          auth_token_generator:
+            implementation: flask_securest.authentication_providers.token:TokenAuthenticator
+            properties:
+              secret_key: my_secret
+              expires_in_seconds: 600
+          bypass_port: 8101
+          ssl:
+            enabled: true
+          audit_log_file: /var/log/cloudify/rest/rest-security-audit.log
```

Comment:
since we're already log rotating the rest's log directory, can we remove this config for now? We'll get to it later when we make all log locations configurable.